must-call-checker/src/main/java/org/checkerframework/checker/mustcall/NotOwning.astub

```diff
@@ -41,7 +41,7 @@ class SocketChannel extends AbstractSelectableChannel implements ByteChannel, Sc
 
 class ServerSocketChannel extends AbstractSelectableChannel implements NetworkChannel {
     protected ServerSocketChannel(SelectorProvider arg0);
-    @MustCallChoice static ServerSocketChannel open(@MustCallChoice ServerSocketChannel this) throws IOException;
+    static ServerSocketChannel open() throws IOException;
```

Comment:
I thought we'd already done this, but it's nbd.

Reply:
It was in a PR that was closed before being merged.